RSP_Requirements_Design_Document/index.html

```diff
@@ -1095,21 +1144,32 @@
 
       <section>
       <h3>Other Operators</h3>
-
+        <p>For completeness, we also include other operators that may result useful in specific use cases. They are of particular interest,
+          especially considering that many use cases require integration with stored data, and even temporally valid stored data.
         <section>
         <h4>Refreshing stored data</h4>
-          <p>Combining streaming and stored data is supported in existing RSP engines. However, in many cases the stored data can change during the query lifetime, so it might be important to refresh or update the stored contents at certain point of the evaluation process timeline. Existing RSP languages do not impose or propose any way of explicitly performing these updates. In fact, in some RSP engines the stored data is assumed to be static during query evaluation.
-An extension to the core functionality of the query language would be for the user to be able to provide hints as to how often the stored data is updated. These may be interpreted by the query processing engine to indicate how often to refresh the stored data. We will need to think about the granularity of these hints, e.g. by dataset, class, etc.</p>
-          <pre class="example highlight" title="An example of this, implemented as a query operator, can be found in <a href="http://code.google.com/p/snee">SNEEql</a>, using the RESCAN keyword:"><code>
+          <p>Combining streaming and stored data is supported in existing RSP engines. However, in many cases the stored data can change during
+            the query lifetime, so it might be important to refresh or update the stored contents at certain point of the evaluation process timeline. 
+            Existing RSP languages do not impose or propose any way of explicitly performing these updates. In fact, in some RSP engines the stored 
+            data is assumed to be static during query evaluation.
+            An extension to the core functionality of the query language would be for the user to be able to provide hints as to how often the stored 
+            data is updated. These may be interpreted by the query processing engine to indicate how often to refresh the stored data. We will need 
+            to think about the granularity of these hints, e.g. by dataset, class, etc. An example of this, implemented as a query operator, can be 
+            found in <a href="http://code.google.com/p/snee">SNEEql</a>, using the RESCAN keyword:
+          </p>
+          <pre class="example highlight" title="RESCAN in SNEEql"><code>
```

Comment:
Sorry if it seems self-adv, but in this RSP 2015 [EoI](https://www.w3.org/community/rsp/files/2015/05/RSP_Workshop_2015_submission_16.pdf) we set some requirements that can help to improve this sub-section.